charts/pega/charts/installer/templates/pega-install-environment-config.yaml

```diff
@@ -62,4 +62,8 @@ data:
   # Distribution kit URL
   DISTRIBUTION_KIT_URL: {{ .Values.distributionKitURL }}
 {{- end }}
-{{ end }}
+  # Bypass loading engine classes into database during installation
+  BYPASS_LOAD_ENGINE_CLASSES: {{ .Values.bypassLoadEngineClasses | quote }}
```

Comment:
If these settings are not present in a values.yaml file (such as when a client pulls the latest version of the charts into a working deployment workspace), will this work?  What will the defaults be, `""`?

Reply:
As I know it should be an error.

Reply:
I don't think this should be an error if the values are not present.  We should just be loading the classes from the DB.  Although I'd prefer having a this default to false vs "", I think an empty string will have the same effect.

Reply:
Yes, it wont error out and I can make it as empty strings by default.

Reply:
> I don't think this should be an error if the values are not present. We should just be loading the classes from the DB. Although I'd prefer having a this default to false vs "", I think an empty string will have the same effect.

I made default as empty strings now.